src/main/groovy/org/standardout/gradle/plugin/platform/internal/FileBundleArtifact.groovy

```diff
@@ -134,13 +134,13 @@ class FileBundleArtifact implements BundleArtifact {
 	 */
 	FileBundleArtifact(BundleArtifact bundle, File sourceBundleFile) {
 		this.file = sourceBundleFile
-		this.id = sourceBundleFile as String
+		this.id = bundle.id + ':sources'
 		this.source = true
 		this.wrap = false // wrapping is done implicitly
 		
 		bndConfig = null
 		
-		version = modifiedVersion = bundle.version
+		version = modifiedVersion = VersionUtil.toOsgiVersion(bundle.version).toString()
```

Comment:
I think if we use the `modifiedVersion` of the bundle directly (instead of adapting the original version here) we should be safe, and the version should be in sync with any changes that are done to the artifact version manually.